File: Model-View-Controller Application.xctemplate/MainView.m
//
// MainView.m
//

#import "MainView.h"
#import "StyleSheet.h"

@interface MainView (){
    
}
@end

@implementation MainView

- (instancetype)initWithFrame:(CGRect)frame {
    if(self = [super initWithFrame:frame]) {
        [self setup:frame];
    }
    return self;
}
- (void)setup:(CGRect)frame {
    self.backgroundColor = [UIColor whiteColor];
    
    _labelHelloWorld = [UILabel new];
    _labelHelloWorld.frame = CGRectMake(0, 0, frame.size.width, margin);
    _labelHelloWorld.center = CGPointMake(frame.size.width/2,  frame.size.height/2);
    _labelHelloWorld.textAlignment = NSTextAlignmentCenter;
    _labelHelloWorld.font = [UIFont systemFontOfSize:36.0];
    [_labelHelloWorld setTextColor:[UIColor blueColor]];
    [self addSubview:_labelHelloWorld];

}

@end

File: Model-View-Controller Application.xctemplate/MainView.swift
//
// MainView.swift
//

import UIKit

class MainView: UIView {

    let labelHelloWorld = UILabel()
    
    override init(frame: CGRect) {
        super.init(frame: frame)
        setup(frame)
    }
    
    required init?(coder aDecoder: NSCoder) {
        super.init(coder: aDecoder)
    }
    
    func setup(frame: CGRect){
        backgroundColor = UIColor.whiteColor()
        
        labelHelloWorld.frame = CGRectMake(0, 0, frame.width,  Global.sharedInstance.margin)
        labelHelloWorld.center = CGPointMake(frame.width/2,frame.height/2)
        labelHelloWorld.textAlignment = NSTextAlignment.Center
        labelHelloWorld.font = UIFont.systemFontOfSize(36.0)
        labelHelloWorld.textColor = UIColor.redColor()
        self.addSubview(labelHelloWorld)
    }
    
}